brave/browser-laptop
# open background page devtools when navigating to background page url

fix #7880
requires https://github.com/brave/muon/commit/42b9c96af6652ee7387d1e26d45a5832d23027da

note: url will not properly reset until #7894 is fixed


Test Plan:
1. Navigate to a background page url (chrome-extension://cjnmeadmgmiihncdidmfiabhenbggfjm/_generated_background_page.html)
2. page should not navigate
3. devtools should open for the background page
4. click on the main window to put the devtools in the background
5. navigate to the background page url again
6. devtools should focus
7. type `console.log('test')` in the devtools window
7. close the devtools
8. navigate to the background page again
9. devtools should open and display the previous output from `console.log('test')`

cc @jonathansampson @bbondy  - please note the muon requirement for this
